fgpyo/util/metric.py

```diff
@@ -226,7 +226,7 @@ def parse(cls, fields: List[str]) -> Any:
         return inspect.attr_from(cls=cls, kwargs=dict(zip(header, fields)), parsers=parsers)
 
     @classmethod
-    def write(cls, path: Path, *values: MetricType) -> None:
+    def write(cls, path: Path, *values: "Metric") -> None:
```

Comment:
Why loose this type hint?  It only writes metrics of this type, so should it not be `MetricType`?

Reply:
Add the bound on `MetricType` to have `Metric`